pyvista/core/filters/data_set.py

```diff
@@ -210,6 +215,9 @@ def _get_quarter(dmin, dmax):
             xmin, xmax, ymin, ymax, zmin, zmax = self.bounds
             bounds = (xmin, xmin+bounds[0], ymin, ymin+bounds[1], zmin, zmin+bounds[2])
         alg = _vtk.vtkBoxClipDataSet()
+        if not merge_points:
+            # vtkBoxClipDataSet uses vtkMergePoints by default
+            alg.SetLocator(_vtk.vtkNonMergingPointLocator)
```

Comment:
Unit test failure suggests these need to be instantiated.
```suggestion
            alg.SetLocator(_vtk.vtkNonMergingPointLocator())
```